src/App.tsx

```diff
@@ -37,12 +37,27 @@ const App = () => {
 
 	const backgroundColor = urlParams.get('bg') || process.env.DESCOPE_BG_COLOR;
 
+	const isWideContainer =
+		urlParams.get('wide') === 'true' || flowId === 'saml-config';
+
+	const containerClassName = isWideContainer ? 'descope-wide-container' : '';
+
+	const flowProps = {
+		flowId,
+		debug,
+		tenant: tenantId,
+		...(isWideContainer && { autoFocus: false })
```

Comment:
```suggestion
		...(flowId === 'saml-config' && { autoFocus: false })
```

I would not bind autoFocus to wide container, but specific to saml config flow, I can see cases where we suggest wide view to "regular" flow, or other flows